dockerfiles/cf-deployment-concourse-tasks/Dockerfile

```diff
@@ -113,7 +113,7 @@ RUN \
 
 RUN \
   go install github.com/cloudfoundry/uptimer@${uptimer_version} && \
-  go install github.com/onsi/ginkgo/...
+  go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
Why include the `-mod=mod` build flag? I wouldn't expect there to be a `go.mod` file or `vendor` directory where this command is run so what's the point of explicitly ignoring them?